src/components/PostGridItem/PostGridItem.js

```diff
@@ -39,21 +42,33 @@ export default function PostGridItem (props) {
   return (
     <div styleName={cx('post-grid-item-container', { unread, expanded }, attachmentType)} onClick={showDetails}>
       <div styleName='content-summary'>
+        {childPost &&
+          <div
+            styleName='icon-container'
+            data-tip='Post from Subgroup'
+            data-for='subgroup-tt'
+          >
+            {/* TODO: i18n on tooltip */}
+            <Icon
+              name='Subgroup'
+              styleName='icon'
+            />
+            <Tooltip
+              delay={250}
+              id='subgroup-tt'
+            />
+          </div>}
         <h3 styleName='title'>{title}</h3>
 
         {/*  Will fix this after I get attachment variables */}
+        {/* TODO: what was meant to happen here? */}
```

Comment:
event dates it looks like?

Reply:
I think Aaron was planning to come back to this and see if he could jimmy dates into the gridItem. its hard, not much space tho